tests/unit/testConfigs/locConstraintAssert.js

```diff
@@ -0,0 +1,118 @@
+import assert from 'assert';
+import { locationConstraintAssert } from '../../../lib/Config';
+
+function generateConfig(testKey) {
```

Comment:
After more discussion, creating a LocationConstraints class and passing it objects mimicking the config with item to change